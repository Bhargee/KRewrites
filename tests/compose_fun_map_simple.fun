"foldl" (fun x -> fun y -> x + y) ("map" (fun a -> a * 3) "map" (fun b -> b - 3))
