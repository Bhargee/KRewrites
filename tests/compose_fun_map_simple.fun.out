<T>
    <k>
        "foldl" (fun (#symExp(2) -> ((fun (x -> (fun (y -> (x + y))))) ((
          fun (#symExp(1) -> ((fun (a -> (a * 3))) ((fun (b -> (b - 3))) 
          #symExp(1))))) #symExp(2)))))
    </k>
    <store>
        .Map
    </store>
    <env>
        .Map
    </env>
</T>
